packs/feats/fey-ascension.json

```diff
@@ -28,7 +28,70 @@
             "remaster": false,
             "title": "Pathfinder Lost Omens: Impossible Lands"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "ActiveEffectLike",
+                "mode": "upgrade",
+                "path": "system.proficiencies.spellcasting.rank",
+                "predicate": [
+                    {
+                        "or": [
+                            "fey-influence:anteater",
+                            "fey-influence:gremlin"
+                        ]
+                    }
+                ],
+                "value": 2
+            },
+            {
+                "category": "unarmed",
+                "damage": {
+                    "base": {
+                        "damageType": "slashing",
+                        "dice": 1,
+                        "die": "d6"
+                    }
+                },
+                "group": "dart",
+                "img": "icons/magic/nature/leaf-oak-glow-green.webp",
+                "key": "Strike",
+                "label": "PF2E.BattleForm.Attack.Leaves",
+                "predicate": [
+                    "fey-influence:dryad"
+                ],
+                "range": {
+                    "increment": 20
+                },
+                "traits": [
+                    "magical",
+                    "unarmed"
+                ]
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "upgrade",
+                "path": "system.skills.med.rank",
+                "predicate": [
+                    "fey-influence:unicorn"
+                ],
+                "value": 1
+            },
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "fey-influence:unicorn",
+                    {
+                        "or": [
+                            "charm",
```

Comment:
Not sure whether to leave to leave the "charm" option for the predicate since it's not really a trait any longer.

Reply:
remove it